arm/subscriptions/client.go

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 0.11.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package subscriptions
+
+import (
+	"fmt"
+	"github.com/azure/go-autorest/autorest"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+const (
+	ApiVersion             = "2014-04-01-preview"
+	DefaultBaseUri         = "https://management.azure.com"
+	DefaultPollingDuration = 10 * time.Minute
```

Comment:
@brendandixon I'm recalling all my comments in this thread. For some reason I read `PollingDuration` as `PollingInterval` and 10 minutes sounded too much. Our long tail in resources like storage account or VMs can sometimes take up to 25-30 minutes and yet would succeed.

So my new comment now is: 10 minutes is too short for most VMs to be created. :smile: 

p.s. sorry for the misunderstanding here.